tuner/examples/test/conv_benchmark.mlir

```diff
@@ -0,0 +1,68 @@
+module {
```

Comment:
These benchmark files should be generated from high-level MLIR inputs. Leaving a comment so that we don't forget about this.

Reply:
Yeah, I'm planning to clean up the example when the rest of the tuner pipeline is landed.